src/Compilers/Core/CodeAnalysisTest/Analyzers/AnalyzerFileReferenceAppDomainTests.cs

```diff
@@ -43,6 +44,7 @@ public void TestAnalyzerLoading_AppDomain()
             var dir = Temp.CreateDirectory();
             dir.CopyFile(typeof(AppDomainUtils).Assembly.Location);
             dir.CopyFile(typeof(RemoteAnalyzerFileReferenceTest).Assembly.Location);
+            dir.CopyFile(typeof(Vector).Assembly.Location);
```

Comment:
This test would fail on line 54. Exception shows it can't find System.Numerics assembly.